server/filters/spacing-classes.js

```diff
@@ -1,6 +1,11 @@
-export default function spacingClasses(sizes, properties) {
-  return Object.keys(sizes).map(key => {
-    const size = sizes[key];
+import { getSizesAsObject } from './utils';
+
+export default function spacingClasses(sizes, properties, fillIn = true) {
```

Comment:
[As mentioned above](https://github.com/wellcometrust/wellcomecollection.org/pull/1167/files#r127421467), I still think it would be simpler / more conventional to just use the cascade?

But happy for that to be another discussion.